src/backend/distributed/planner/multi_master_planner.c

```diff
@@ -134,6 +134,7 @@ BuildAggregatePlan(Query *masterQuery, Plan *subPlan)
 	Assert(masterQuery->hasAggs || masterQuery->groupClause);
 
 	aggregateTargetList = masterQuery->targetList;
+	memset(&aggregateCosts, 0, sizeof(AggClauseCosts));
```

Comment:
I see `memset` a lot in our code, but have been trying to use `MemSet` myself, due to its purported benefits. Is this meaningful? If so, change this occurrence I guess.